tests/fastboot/nock-proxy-test.js

```diff
@@ -0,0 +1,95 @@
+import { module, test } from 'qunit';
+import { setup, visit, nock } from 'ember-cli-fastboot-testing/test-support';
+
+module('Fastboot | nock proxy', function(hooks) {
+  setup(hooks);
+
+  test('it will not change an endpoint that already exists', async function(assert) {
+    await visit('/examples/network/other/echo?message=hello%20world');
+    assert.dom('[data-test-id="echo"]').hasText("hello world");
+  });
+
+  test('it can mock an array of models', async function(assert) {
+    await nock('http://localhost:7357')
```

Comment:
Is it possible to auto detect the local hostname and not require it here?

Reply:
We could set the `hostname` in the application adapter, and use that here. It wouldn’t then change when, for example, using a different testing port.